src/audio/pcm_converter/pcm_converter_generic.c

```diff
@@ -134,24 +134,26 @@ const struct pcm_func_map pcm_func_map[] = {
 #if CONFIG_FORMAT_S16LE
```

Comment:
Can you please rewrite this commit message. It can be made much more clear with re-wording. Thanks!